README.md

```diff
@@ -1,5 +1,12 @@
 # esignet-plugins
 Repository hosting source code for esignet and esignet-signup java runtime dependencies plugins.
 
+## eSignet/esignet-signup service image bundled with plugins
+We release esignet service and esignet-signup service images with the latest published plugin jars  
+available at the time of the esignet and esignet-signup service release.
+
+Any bug fixes or changes in the plugins will be made available in the esignet service or 
+esignet-signup service in their immediate next releases.
```

Comment:
lets also call out how to use the plugin with fixes where the eSignet is not yet released. so QA can also follow the same for their testing of the fixes.